src/OrchardCore.Build/Dependencies.props

```
@@ -46,8 +46,8 @@
     <PackageManagement Include="NJsonSchema" Version="10.9.0" />
     <PackageManagement Include="NLog.Web.AspNetCore" Version="5.3.2" />
     <PackageManagement Include="NodaTime" Version="3.1.9" />
-    <PackageManagement Include="OpenIddict.AspNetCore" Version="4.6.0" />
-    <PackageManagement Include="OpenIddict.Core" Version="4.6.0" />
+    <PackageManagement Include="OpenIddict.AspNetCore" Version="4.7.0-preview1.23377.63" />
+    <PackageManagement Include="OpenIddict.Core" Version="4.7.0-preview1.23377.63" />
```

```suggestion
    <PackageManagement Include="OpenIddict.AspNetCore" Version="4.7.0" />
    <PackageManagement Include="OpenIddict.Core" Version="4.7.0" />
```

https://github.com/openiddict/openiddict-core/releases/tag/4.7.0